earth_enterprise/src/fusion/autoingest/Asset.h

```diff
@@ -36,27 +36,27 @@
  ***  ... = asset->config.layers.size();
  ***
  ******************************************************************************/
-class AssetImpl : public khRefCounter, public AssetStorage {
+class AssetImpl : public khRefCounter, public AssetStorage, public StorageManaged {
   friend class AssetHandle_<AssetImpl>;
 
   // private and unimplemented -- illegal to copy an AssetImpl
   AssetImpl(const AssetImpl&);
   AssetImpl& operator=(const AssetImpl&);
```

Comment:
maybe better to delete, along with moves also

Reply:
Good catch. Fixed.